### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by MadhuAtBerkeley** [TensorSpeech/TensorFlowASR#152]
Hi @usimarit, 
  I trained the conformer (default config) only for train-clean-100 (Librispeech) files for 20 epochs and wanted to check if everything is working as expected.  The loss curve is attached below.   I see that decoded out is wrong - it is same text "and the other of the others were"  

Please let me know how to resolve this issue. It looks like a decoder issue and I am using default config file with beam_width = 0 (greedy). I hope 20 epochs is enough to get a decent prediction with train-clean-100 and test-clean corpus.

Thanks

PATH	GROUNDTRUTH	GREEDY	BEAMSEARCH	BEAMSEARCHLM
/media/user1/My4TBHD1/transformer/TensorFlowASR/LibriSpeech/test-clean/5683/32866/5683-32866-0000.flac	miss lake declined the carriage to night	and the other of the others were		
/media/user1/My4TBHD1/transformer/TensorFlowASR/LibriSpeech/test-clean/5683/32866/5683-32866-0001.flac	and he added something still less complimentary	and the other of the others were		
/media/user1/My4TBHD1/transformer/TensorFlowASR/LibriSpeech/test-clean/5683/32866/5683-32866-0002.flac	but don't these very wise things sometimes turn out very foolishly	and the other of the others were

![loss](https://user-images.githubusercontent.com/64989692/108895749-d86f1680-75c8-11eb-8494-2cb7f77c5a48.png)

**Comment by MadhuAtBerkeley** [comment]
Also, I would like to verify the conformer model with a small set of train/dev/test data.   Is it possible to use 10-15 hours of clean train data and verify small test data?   Thanks

**Comment by usimarit** [comment]
The loss is too high, the model doesn't learn anything yet.
And it might be because of the issue #153 too

**Comment by MadhuAtBerkeley** [comment]
Okay. I will use entire Librispeech data (100,360 and 500 hours) for training then.  i see keras training version is bit faster.  However, the train module does not seem to save the model (latest.h5) with default config file. Anything extra needed?  Thanks

**Comment by usimarit** [comment]
@MadhuAtBerkeley please see the config and keras training script in the example, you will see the details

**Comment by MadhuAtBerkeley** [comment]
@usimarit I do not see any h5 files - checkpoints/{epoch:02d}.h5.  I suspect some interaction between ModelCheckpoint and BackupAndRestore callbacks in TF 2.4.1.

**Comment by MadhuAtBerkeley** [comment]
@usimarit I found the reason - the default monitor is monitor='val_accuracy' for ModelCheckpoint() and need to pass monitor='val_val_rnnt_loss' for conformer.  Also, good to change {"val_rnnt_loss": loss} in transducer to 'rnnt_loss'  so that loss becomes 'val_rnnt_loss' (currently printed as val_val_rnnt_loss).

Let me know if I should generate a pull request (I might end up making more changes)

train_rnnt_loss: 3118.5235 - val_val_rnnt_loss: 3056.2104

**Comment by usimarit** [comment]
@MadhuAtBerkeley I already changed (config.yml not updated), it's available in v0.7.7 and above
